### PR TITLE
add new solution

### PR DESCRIPTION
**Pull request by ghfvs-tester**

### CODE REVIEW

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:7]: cool

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyTitle("WindowsFormsApplication1")]
+[assembly: AssemblyDescription("")]
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:9]: !!@@3lgtm

**Review by ghfvs-tester — COMMENTED** [review]
can i comment?

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx [hunk 3]
```diff
@@ -0,0 +1,117 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:6]: test comment

Comment (meaghanlewis) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.resx:6]: reply

**Review by meaghanlewis — APPROVED** [review]
lalala

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("WindowsFormsApplication1")]
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:8]: <3

**Review by meaghanlewis — APPROVED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Settings.settings [hunk 5]
```diff
@@ -0,0 +1,7 @@
+﻿<?xml version='1.0' encoding='utf-8'?>
+<SettingsFile xmlns="http://schemas.microsoft.com/VisualStudio/2004/01/settings" CurrentProfile="(Default)">
+  <Profiles>
+    <Profile Name="(Default)" />
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Settings.settings:4]: comment!!!1 >?<<>>>

**Review by meaghanlewis — APPROVED** [review]

**Review by meaghanlewis — APPROVED** [review]
lgtm!!!!!!!

**Review by meaghanlewis — CHANGES_REQUESTED** [review]
nope

File: WindowsFormsApplication1/WindowsFormsApplication1/App.config [hunk 6]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/App.config:4]: 1

File: WindowsFormsApplication1/WindowsFormsApplication1/App.config [hunk 7]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/App.config:2]: 2

File: WindowsFormsApplication1/WindowsFormsApplication1/App.config [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+    </startup>
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/App.config:5]: 3

File: WindowsFormsApplication1/WindowsFormsApplication1.sln [hunk 9]
```diff
@@ -0,0 +1,22 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
+VisualStudioVersion = 14.0.25420.1
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WindowsFormsApplication1", "WindowsFormsApplication1\WindowsFormsApplication1.csproj", "{CCFAF382-6679-4ECA-A84F-7D4C64FA2F9F}"
+EndProject
+Global
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1.sln:8]: 4

**Review by meaghanlewis — APPROVED** [review]

**Review by meaghanlewis — APPROVED** [review]
LGTM :thumbs_up:

**Review by meaghanlewis — CHANGES_REQUESTED** [review]
nope

File: WindowsFormsApplication1/WindowsFormsApplication1/Form1.cs [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace WindowsFormsApplication1
+{
+	public partial class Form1 : Form
+	{
+		public Form1()
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Form1.cs:15]: comment from editor

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace WindowsFormsApplication1.Properties
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs:11]: start a review

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+{
+
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/Resources.Designer.cs:13]: kakaijfoa

**Review by meaghanlewis — APPROVED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (meaghanlewis) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:7]: reply

**Review by meaghanlewis — DISMISSED** [review]

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("WindowsFormsApplication1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:13]: add a new comment

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```
Comment (meaghanlewis) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:15]: another one?

**Review by ghfvs-tester — COMMENTED** [review]
new comment

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("ccfaf382-6679-4eca-a84f-7d4c64fa2f9f")]
+
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:24]: new comment

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: comment

**Review by ghfvs-tester — COMMENTED** [review]
new comment

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("WindowsFormsApplication1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:18]: new comment

Comment (ghfvs-tester) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs:18]: new reply

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("ccfaf382-6679-4eca-a84f-7d4c64fa2f9f")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: comment

Comment (ghfvs-tester) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: reply

File: WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+[assembly: Guid("ccfaf382-6679-4eca-a84f-7d4c64fa2f9f")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
```
Comment (ghfvs-tester) [inline WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: coment

Comment (ghfvs-tester) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: reply

Comment (ghfvs-tester) [reply WindowsFormsApplication1/WindowsFormsApplication1/Properties/AssemblyInfo.cs]: reply and start revie

**Review by meaghanlewis — APPROVED** [review]

**Review by meaghanlewis — COMMENTED** [review]
comment

**Pull request merged**